feature/experimental/isis/otg_tests/base_adjacencies_test/README.md

```diff
@@ -6,39 +6,22 @@ Base IS-IS functionality and adjacency establishment.
 
 ## Procedure
 
-*   Basic fields test
```

Comment:
What's the reason for the doc changes? (AFAIK, I think they should capture the intent of the test, not necessarily the current state.)

Reply:
@dplore : ^ Is my understanding correct there?

Reply:
Yes, the readme should indicate the intent of the test.  If some of the intent is not implemented, we should note it in the readme with TODO.     

Looks like this proposes to remove coverage for hello padding?  (ie: only test with hello padding disabled?)

Reply:
@octpetre : I think just add the TODO's to each section of the procedure with a note on the current deviation. (So keep the text largely the same.)

Reply:
Updated the README to match the ATE version